lib/puppet-lint/plugins/top_scope_facts/top_scope_facts.rb

```diff
@@ -0,0 +1,35 @@
+# Public: A puppet-lint plugin that will check for the use of top scope facts.
```

Comment:
should the file contain a link to the original author and repo?

Reply:
Resolved